middlewares/isAuthorized.js

```diff
@@ -0,0 +1,28 @@
+import Listing from "../schemas/listingSchema.js";
+import Review from "../schemas/reviewSchema.js";
+
+const isListingAuthor = async (req, res, next) => {
```

Comment:
name of this function is not intutive.. the name should `isUserListingOwner`

Reply:
done